### ISSUE TITLE
Did you freeze any layer in the fine-tune step for the results reported in simclr-v2 paper?

### ISSUE THREAD

**Issue by cp1123** [google-research/simclr#109]
Hi, there. I wonder for the results reported in the paper, did you fine-tune the whole network, meaning the flag fine_tune_after_block == -1? From the paper, it says "For fine-tuning, by default we fine-tune from the first layer of the projection head for 1%/10% of labeled examples, but from the input of the projection head when 100% labels are present. " so I initially thought you only fine-tuned the weights after this layer (meaning freezing all the layers beforehand) but it seems contradicting the rest of the paper and the codebase. Please help clarify. Thanks!

**Comment by chentingpc** [comment]
Yes, the fine-tuning means fine-tuning the whole network. The description of " fine-tune from the first layer of the projection head" means we keep the first layer of projection head and add a linear head with softmax on top of that when fine-tuning.

Hope that makes it clear.

**Comment by cp1123** [comment]
Thank you for clarification!
